src/main/java/edu/ie3/datamodel/io/extractor/Node.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.extractor;
+
+import edu.ie3.datamodel.models.input.NodeInput;
+
+/**
+ * Interface that should be implemented by all elements holding a {@link NodeInput} and should be
+ * processable by the {@link Extractor}
+ *
+ * @version 0.1
+ * @since 31.03.20
+ */
+public interface Node extends Nested {
```

Comment:
What about pimpin the `Nodes` interface and letting go `Node` and `NodeC`? See comments below.